mobile/ios/Interface.mm

```diff
@@ -65,6 +65,9 @@ + (NSArray*)iota_ios_pow_bundle:(NSArray*)txsTrytes trunk:(NSString*)trunk branc
   NSMutableArray* outputTxsTrytes = [NSMutableArray array];
   NSMutableString* outputTxsTrytesSerialized = [NSMutableString string];
   uint8_t cmwm = (uint8_t)[mwm unsignedCharValue];
+    bundle_miner_ctx_t* ctxs = NULL;
+    size_t num_ctxs = 0;
+    bool found_optimal_index = false;
```

Comment:
This seems like it was added to `iota_ios_pow_bundle` instead of `bundle_miner_mine`